src/bootstrap/src/utils/helpers.rs

```diff
@@ -272,6 +272,15 @@ pub fn output(cmd: &mut Command) -> String {
     String::from_utf8(output.stdout).unwrap()
 }
 
+#[track_caller]
+pub fn try_output(cmd: &mut Command) -> String {
+    let output = match cmd.stderr(Stdio::inherit()).output() {
+        Ok(status) => status,
+        Err(e) => fail(&format!("failed to execute command: {cmd:?}\nERROR: {e}")),
+    };
+    String::from_utf8(output.stdout).unwrap_or("".to_string())
```

Comment:
Please just panic or similar if the output isn't utf-8. Returning an empty String is confusing.